docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -60,10 +62,15 @@ The properties specific to IBFT 2.0 are:
 * `blockperiodseconds` - The minimum block time, in seconds.
 * `epochlength` - The number of blocks after which to reset all votes.
 * `requesttimeoutseconds` - The timeout for each consensus round before a round change, in seconds.
+* `blockreward` - Optional reward amount in Wei to reward the beneficiary. Defaults to zero (0).
+    Can be specified as a hexidecimal or decimal string value.
```

Comment:
is it worth saying hexidecimal (with 0x prefix) or decimal string value?

Reply:
Done